tests/integration/SecurityUpdatesTest.php

```diff
@@ -23,13 +23,6 @@ public function testInsecureDrupalPackage()
         $this->arrayHasKey($expected_package, $security_advisories);
         $this->assertEquals($expected_package, $security_advisories[$expected_package]['name']);
         $this->assertEquals($expected_version, $security_advisories[$expected_package]['version']);
-        // Remove this clause if we don't have an insecure release to use.
```

Comment:
I'm not too happy about losing this test coverage. If we have to lose it for Drupal 9.1 thats fine, but losing it for all our Druapals seems like a bad idea. This has caught bugs before. I'm not pacified by the fact that contrib remains tested.

Reply:
It is bad, though, to pin all of our Drupal 9 tests to 9.0.0, as then we don't catch any changes to Drupal 9. 

I'll push one more iteration & see what you think.